pycontrails/core/met.py

```diff
@@ -2248,6 +2247,26 @@ def shift_longitude(data: XArrayType) -> XArrayType:
     ).sortby("longitude", ascending=True)
 
 
+def shift_longitude_reverse(data: XArrayType) -> XArrayType:
```

Comment:
It's a little more work, but you could combine the two shift longitude functions into one. Something like this:

```python
def shift_longitude_more_general(data: XArrayType, a: float) -> XArrayType:
    return data.assign_coords(
        longitude=((data["longitude"].values + a) % 360.0) - a
    ).sortby("longitude", ascending=True)
```

This would shift the longitude from any domain to the interval `[-a, 360 - a)`. Then your new `shift_longitude_reverse` is the same as evaluating this one with `a=0.0`. And the current `shift_longitude` is the same as evaluating this one with `a=180.0`.

Reply:
Fantastic suggestion, simplified and included in 4e7fbf3cea7942b56b4255ffdd4be4f607406c94